src/Hook.php

```diff
@@ -22,10 +22,19 @@ public static function getValidHooks()
         );
         $validHooks = [];
 
+        //add customize hooks
+        $customizeHooks = isset($json['extra']['customize_hooks']) ?
```

Comment:
I don't think so. This  project is order to create some scripts for git, some scripts been known as git hooks. So, i think it can add custom hooks. Thanks.